### PR TITLE
BAEL-5990 is implemented to generate JMeter dashboard reports

### PR DESCRIPTION
**Pull request by muasif80**

### CODE REVIEW

File: jmeter/pom.xml [hunk 1]
```diff
@@ -68,5 +68,91 @@
     <properties>
         <jmeter.version>2.6.0</jmeter.version>
     </properties>
+    
+    <profiles>
+		<profile>
```
Comment (andrzejsydor) [inline jmeter/pom.xml:73]: the formatting is weird, please try reformat code

File: jmeter/src/main/java/com/baeldung/dashboard/controllers/DashBoard.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        
+        int wait = r.nextInt(6);
+        Thread.currentThread().sleep(wait);
+        model.addAttribute("quote", quote);
+        
+        return "quote";
+    }
+    
+    @GetMapping("/time")
+    public String getTime(Model model) throws InterruptedException {
```
Comment (andrzejsydor) [inline jmeter/src/main/java/com/baeldung/dashboard/controllers/DashBoard.java:36]: why not use: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/ModelAndView.html

```suggestion
    public String getTime(ModelAndView model) throws InterruptedException {
```

**Review by andrzejsydor — CHANGES_REQUESTED** [review]
comments added

File: jmeter/src/main/java/com/baeldung/dashboard/controllers/DashBoard.java (hunk 2 above)
Comment (muasif80) [reply jmeter/src/main/java/com/baeldung/dashboard/controllers/DashBoard.java:36]: Its a matter of preference. Right now its just a demo app to create some endpoints. I consider use of Model ok here. Thanks.

**Pull request merged**